web-ts/jest.setup.ts

```diff
@@ -28,6 +24,7 @@ function walkDir(dir: string, callback: (dir: string) => void) {
 // modules automatically as soon as they are found
 const mocksPath = 'src/__mocks__/'
 const jsPath = '.js' // This only unmocks .js files not .json for example
+
 walkDir(mocksPath, name => {
   if (name.endsWith(jsPath)) {
     jest.unmock(name.substring(mocksPath.length, name.length - jsPath.length))
```

Comment:
This should be adjusted the same way as in the native project. Otherwise jest will automatically mock everything with a file in __mocks__